code/modules/food_and_drinks/food.dm

```diff
@@ -1,19 +1,67 @@
 ////////////////////////////////////////////////////////////////////////////////
 /// Food.
 ////////////////////////////////////////////////////////////////////////////////
+GLOBAL_LIST_INIT(temp_stabilizers, typecacheof(list(
+	/obj/structure/closet/crate/secure/hydroponics,
+	/obj/structure/closet/crate/freezer,
+	/obj/structure/closet/crate/hydroponics,
+	/obj/structure/closet/secure_closet/freezer
+	)))
+
 /obj/item/reagent_containers/food
 	possible_transfer_amounts = list()
 	volume = 50	//Sets the default container amount for all food items.
 	container_type = INJECTABLE_1
 	resistance_flags = FLAMMABLE
 	var/foodtype = NONE
 	var/last_check_time
+	var/cares_about_temperature = FALSE
+	var/starting_temp = 200 // Farenheight
```

Comment:
Kelvin is absurd to be using for this, I just chatted with buddies working in fine dining, one's a Sous Chef, other's a Head Chef

they both laughed their asses off about this because Kelvin is never ever used in culinary and I have no plans to make chemical temp/atmos temp interact with food temp as that'd just make all the shit I'm doing here easy to powergame.

Fahrenheit and Celsius are the two temps used in Culinary, and it's going to stay as Fahrenheit for accuracy and realism, and because part of the goal here is realism and ability to translate some of these skills back and forth between vidya and IRL. Ask anyone who works in the food industry.